doc/source/release-notes/Percona-Server-5.7.25-28.rst

```diff
@@ -0,0 +1,72 @@
+.. rn:: 5.7.25-28
+
+================================================================================
+|Percona Server| |release|
+================================================================================
+
+Percona is glad to announce the release of |Percona Server| |release| on
+|date|. Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.25-28/>`_
+and from the :doc:`Percona Software Repositories </installation>`.
+	
+This release is based on `MySQL 5.7.25
+<http://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-25.html>`_ and includes
+all the bug fixes in it. |Percona Server| |release| is now the current GA
+(Generally Available) release in the 5.7 series.
+	
+All software developed by Percona is open-source and free.
+	
+.. note:: 
+
+   If you're currently using |Percona Server| 5.7, Percona recommends upgrading
+   to this version of 5.7 prior to upgrading to |Percona Server| 8.0.
+
+Bugs Fixed
+================================================================================
+
+- FLUSH commands written to the binary log could cause errors in case
+  of replication. Bug fixed :psbug:`1827`: (upstream :mysql:`88720`).
+- Running LOCK TABLES FOR BACKUP followed by STOP SLAVE SQL_THREAD
+  could block replication preventing it from being restarted
+  normally. Bug fixed :psbug:`4758` (upstream :mysqlbug:`93649`).
+- InnoDB did not report the filename in I/O error messages. Bug fixed
+  :psbug:`2455`.
+- The ``ACCESS_DENIED`` field of the
+  information_schema.user_statistics table was not updated
+  correctly. Bug fixed :psbug:`3956`.
+- MySQL could report that the maximum number of connections was
+  exceded with too many connections being in the CLOSE_WAIT state. Bug
+  fixed :psbug:`4716` (upstream :mysqlbug:`92108`)
+- ``http`` was replaced with ``https`` in http://bugs.percona.com in
+  server crash messages. Bug fixed :psbug:`4855`.
+- Wrong query results could be received in semi-join sub queries with
+  materialization-scan that allowed inner tables of different
+  semi-join nests to interleave. Bug fixed :psbug:`4907` (upstream bug
+  :mysqlbug:`92809`).
+- :psbug:`4911`: In some cases, the server using the the MyRocks
+  storage engine could crash when TTL (Time to Live) was defined on a
+  table.
+- In some ccases, running the SELECT stament with the ORDER BY and
+  LIMIT clauses could result in a less than optimal performance. Bug
+  fixed :psbug:`4949` (ustream :mysql:`92850`)
+- There was a typo in ``mysqld_safe.sh``: **trottling** was replaced
+  with **throttling**. Bug fixed :psbug:`240`. Thanks to Michael
+  Coburn for the patch.
+
+Other bugs fixed:
+:psbug:`4705`,  
+:psbug:`4791`, 
+:psbug:`4797`,
+:psbug:`4958`,
```

Comment:
Check with @dutow if this one needs description (it's a server not testsuite bug)

Reply:
Does it?